src/data/majors/ns.ts

```diff
@@ -0,0 +1,280 @@
+import { CollegeOrMajorRequirement, Course } from '../../requirements/types';
+import {
+  courseIsFWS,
+  hasCategory,
+  ifCodeMatch,
+  includesWithSubRequirements,
+} from '../../requirements/checkers';
+import { AdvisorGroup } from '../../tools/advisors/types';
+
+const nsSocialSciencesHumanities: readonly string[] = ['CA', 'D', 'FL', 'HA', 'KCM', 'LA', 'SBA'];
+
+const nsRequirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: 'Introductory Chemistry',
+    description: 'Must take both CHEM 2070 & CHEM 2080.',
+    source: 'https://courses.cornell.edu/preview_program.php?catoid=55&poid=28314#majorrequire',
+    checker: includesWithSubRequirements(['CHEM 2070'], ['CHEM 2080']),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 1],
+    slotNames: ['CHEM 2070', 'CHEM 2080'],
+  },
+  {
+    name: 'Introductory Biology',
+    description:
+      'Must complete one bio lab and two bio lectures. One of the following labs: BIOG1500, BIOSM1500. Two of the following three lecture options: 1) BIOMG1350 2) BIOG1440 OR BIOG1445 3) BIOEE1610 OR BIOEE1780',
+    source: 'https://courses.cornell.edu/preview_program.php?catoid=55&poid=28314#majorrequire',
+    checker: includesWithSubRequirements(
+      ['BIOG 1500', 'BIOSM 1500'],
+      ['BIOMG 1350', 'BIOG 1440', 'BIOG 1445', 'BIOEE 1610', 'BIOEE 1780']
+    ),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 2],
+    slotNames: ['Introductory Biology Laboratory', 'Introductory Biology Lecture'],
+  },
+  {
+    name: 'Organic Chemistry Lecture',
+    description: 'CHEM 1570 OR CHEM 3570 & 3580 OR CHEM 3590 & 3600 OR CHEM 3530',
+    source: 'https://courses.cornell.edu/preview_program.php?catoid=41&poid=19800',
+    checker: includesWithSubRequirements(
+      ['CHEM 1570'],
+      ['CHEM 3570', 'CHEM 3580'],
+      ['CHEM 3590', 'CHEM 3600'],
+      ['CHEM 3530']
+    ),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 2, 2, 1],
+    slotNames: ['CHEM 1570', 'CHEM 3570 or CHEM 3580', 'CHEM 3590 or CHEM 3600', 'CHEM 3530'],
+    minNumberOfSlots: 1,
+  },
+  {
+    name: 'Organic Chemistry Lab',
+    description: 'Either CHEM 2510 (Intro to Exp Org. Chem) or CHEM 3010 (Honors Exp. Chem I)',
+    source: 'https://courses.cornell.edu/preview_program.php?catoid=55&poid=28314',
+    checker: includesWithSubRequirements(['CHEM 2510', 'CHEM 3010']),
```

Comment:
```suggestion
    checker: includesWithSingleRequirement('CHEM 2510', 'CHEM 3010'),
```
We have a special function for where there is only one slot--can you make this change throughout this file?